terra-env/main.tf

```diff
@@ -72,7 +72,7 @@ module "sam" {
 }
 
 module "workspace_manager" {
-  source = "github.com/broadinstitute/terraform-ap-modules.git//terra-workspace-manager?ref=terra-workspace-manager-0.6.6"
+  source = "github.com/broadinstitute/terraform-ap-modules.git//terra-workspace-manager?ref=jaycarlton%2FPF-810"
```

Comment:
Minor suggestion: personally I did not use `/ ` in branch name, but use `-` instead.

Reply:
old habits die hard